_source/_posts/2020-09-22-reintroducing-joel-franusic.md

```diff
@@ -0,0 +1,331 @@
+
+One example of something I've made is a page that lets
+you [edit the code of the Okta Sign-In Widget with instant feedback, from inside
+your web browser](https://developer.okta.com/live-widget/). However, it uses JavaScript and, since I'm focusing
+on Python and Go, I wanted to make an example of running Python from inside of
+the browser. So, I wrote a game for you to play which I have included below:
+
+
+# Running Python in a Web Browser
+
+This is a simple game where the goal is to write a Python function (`answer()`) that takes
+three numbers of a sequence as input (we label these `a`, `b`, and `c`) and
+returns the next number in that sequence.
+
+For example, let's imagine that the `answer(a, b, c)` function is given part of this
+integer sequence as input:
+
+<p style="height: 150px">
+  <a href="https://dilbert.com/strip/2001-10-25">
+    <img style="max-height: 90%; max-width:90%;" src="https://assets.amuniversal.com/321a39e06d6401301d80001dd8b71c47"/>
+  </a>      
+</p>
+
+In this case, `answer()` would be called with the first three numbers of the sequence, like so:
+`answer(9, 9, 9)`. In this case, the correct answer would be `9` and the code
+to "solve" this sequence might look something like this:
+
+    def answer(a, b, c):
+        return 9
+
+To win this game, you have to write a function that will be able to give
+the correct answer for **all** of the sequences that show up on the
+checklist below. The checklist will update as you solve for each sequence.
+Update the Python code below to start playing!
+
+Sequence checklist:
+
+  <ul id="messages"></ul>
+  <div id="error" class="alert error"></div>
+  <div id="success" class="alert success">
+    You did it!
+    Reward yourself by <a href="https://oeis.org/wiki/Welcome">learning more about The On-Line Encyclopedia of Integer Sequences</a> ®
+    and why Donald Knuth says you can use it to
+    <a href="https://youtu.be/BxQw4CdxLr8?t=1187">"compute your way into the literature"</a>.
+  </div>
+  <textarea id="editor">def answer(a, b, c):
+    return True
+
+play(answer)</textarea>
+<script type="text/javascript">
+document.addEventListener("DOMContentLoaded", function(){
+    brython(1)
+});
+</script>
+
+Edit the code above to play. No need to press a "save" button or anything. Your code will run once you've
+stopped typing for a bit.
+
+
+# "That's interesting, I guess. Why should I care?"
+
+Perhaps you played the game, perhaps you didn't. But in either case, the reason
+I included this game in this blog post might not be apparent. I
+did so to show that, not only is it fairly easy to run
+a non-JavaScript interpreter in the browser, it can be a lot of fun when you do.
+
+I wanted to make something that:
+
+* Involves writing some mildly-complex code, something just a little bit more complicated than "hello world".
+* Runs as soon as you stop typing.
+* Teaches the player something new.
+* Shows that it's fairly easy, and fun, to run Python directly from a browser.
```

Comment:
This list doesn't seem to show up as bullet points in the preview. Not sure why.

<img width="904" alt="Screen Shot 2020-09-21 at 11 39 41 AM" src="https://user-images.githubusercontent.com/17892/93801459-428d2500-fbff-11ea-998d-9c8c5ea657a6.png">

Reply:
I'm not sure why either. I'm assuming we need to fix a style sheet somewhere, but I was planning on doing that after-the-fact. What do you suggest?

Reply:
It looks like it's caused by some embedded CSS you have in this post. 

<img width="1573" alt="Screen Shot 2020-09-21 at 12 48 22 PM" src="https://user-images.githubusercontent.com/17892/93808200-e62f0300-fc08-11ea-930d-a1b478162fca.png">

Reply:
Oh snap! Thanks! I'll fix that right away.

Reply:
Fixed! How's it look now?

Reply:
LGTM!